processor/routingprocessor/documentation.md

```diff
@@ -0,0 +1,31 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# routing
+
+## Internal Telemetry
+
+The following telemetry is emitted by this component.
+
+### routing_processor_non_routed_log_records
```

Comment:
Not a big deal but I noticed the name for these puts the type before the component type. This is the inverse of other processor metrics

Reply:
Changing that would break compatibility with existing consumers of this metric. I'd like to keep it as it is right now and prepare a plan for improving.